components/tools/OmeroPy/src/omero/plugins/metadata.py

```diff
@@ -211,6 +212,16 @@ def _configure(self, parser):
             "--measurement", type=int, default=None,
             help="Index of the measurement to populate. By default, all")
 
+        pixelsize.add_argument(
+            "--x", type=float, default=None, help="Physical pixel size X")
+        pixelsize.add_argument(
+            "--y", type=float, default=None, help="Physical pixel size Y")
+        pixelsize.add_argument(
+            "--z", type=float, default=None, help="Physical pixel size Z")
+        pixelsize.add_argument(
+            "--unit", default="um", help="Unit (e.g. nm, um, mm, ...) "
```

Comment:
If the sizes need different units, the suggestion would be to call the plugin multiple times?